packages/web/src/baklava/LeaderBoard.tsx

```diff
@@ -0,0 +1,56 @@
+import ApolloClient from 'apollo-boost'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import getenv from 'getenv'
+import gql from 'graphql-tag'
+import fetch from 'node-fetch'
+import * as React from 'react'
+import { ApolloProvider, Query } from 'react-apollo'
+import LeaderBoardError from 'src/baklava/LeaderBoardError'
+import LeaderBoardLoading from 'src/baklava/LeaderBoardLoading'
+import LeaderBoardView from 'src/baklava/LeaderBoardView'
+import { I18nProps, NameSpaces, withNamespaces } from 'src/i18n'
+
+// TODO: check how to load this URL from env
+const graphqlURI = getenv('REACT_APP_WS_GRAPHQL', 'http://localhost:8080/v1/graphql')
```

Comment:
this can be placed in env-config file and loaded with with getConfig from next/config